### PR TITLE
NATIVE-8: Update documentation

### PR DESCRIPTION
**Pull request by klinzo**
Also: NATIVE-274: Add note about renaming from `news` to `events`

### CODE REVIEW

File: docs/14-i18n.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The app has two languages:
+* **UI language**
+* **Content language**
+
+## UI language
+
+The UI language is always the primary system language and cannot be changed.
+
+All UI and navigation elements are using the UI language and the corresponding layout direction, i.e. LTR or RTL.
```
Comment (maxammann) [inline docs/14-i18n.md]: We should decide here for a naming of the elements. I think UI elements is fine.

File: docs/14-i18n.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The content language is changeable in the app and is set to the UI language at the first app start.
+
+All content elements are using the content language and the corresponding layout direction, i.e. LTR or RTL.
+Content elements are all elements that are NOT UI elements, especially
+* Webview content
+* Categories, Events, Offers tiles and lists
+* LastUpdate
+
+## WIP: To discuss
```
Comment (maxammann) [inline docs/14-i18n.md]: I would go for this rule:
_If the element is important for a user who is shown the app during counseling, then it is content language. _

Search, Feedback, Language change and Landing page are not important for such users.

Comment (maxammann) [reply docs/14-i18n.md]: We should also finish this here as there shouldn't be an other task imo

Comment (klinzo) [reply docs/14-i18n.md]: What do you mean by `finish this here`? Just  to decide whether the items in `To discuss` are belonging to content or ui? That was the plan!

Comment (klinzo) [reply docs/14-i18n.md]: And I like your rule. @Schedulaar what do you think?

File: docs/14-i18n.md (hunk 1 above)
Comment (Schedulaar) [reply docs/14-i18n.md]: Looking good to me.

**Pull request merged**